src/Mapping/DirectMapping.php

```diff
@@ -14,12 +14,14 @@
  * Class DirectMapping
  * @package FrankHouweling\Cartograph\Mapping
  */
-class DirectMapping implements MappingInterface
+final class DirectMapping implements MappingInterface
```

Comment:
I'm not sure if I want this to be final. It can be usefull to users of the library to extend the directmapping with custom functionallity. Also, this is a backwards compatbility break.